.travis.yml

```diff
@@ -8,6 +8,10 @@ bundler_args: "--without documentation --path bundle"
 before_install:
 - git fetch --depth=1
 - gem install bundler
+- wget --no-check-certificate https://binaries.rightscale.com/rsbin/fpt/v1/fpt-linux-amd64.tgz
+- tar zxvf fpt-linux-amd64.tgz
+- mv fpt testfpt
+- mv testfpt/fpt .
```

Comment:
lets make this a one liner. something like this

mkdir -p tmp && wget --no-check-certificate  https://binaries.rightscale.com/rsbin/fpt/v1/fpt-linux-amd64.tgz  && tar zxvf fpt-linux-amd64.tgz -C tmp;mv tmp/fpt/fpt .